core/src/main/java/io/aiven/klaw/service/UtilControllerService.java

```diff
@@ -596,9 +596,8 @@ public Map<String, String> getBasicInfo() {
     Map<String, String> resultBasicInfo = new HashMap<>();
     resultBasicInfo.put("contextPath", kwContextPath);
 
-    if ("true".equals(ssoEnabled)) {
-      resultBasicInfo.put("ssoServerUrl", ssoServerLoginUrl);
-    }
+    resultBasicInfo.put("ssoServerUrlAzure", ssoServerLoginUrlAzure);
+    resultBasicInfo.put("ssoServerUrlGoogle", ssoServerLoginUrlGoogle);
```

Comment:
These might be null, right?

Reply:
No, we have configured them under the unmodifiable section in the app properties like below.

https://github.com/aiven/klaw/pull/343/files#diff-3a4af74a2e7d8579605a1f3ff6719d674faa6a0e3f989071d9f81d41292043f2R166